config/config-server/src/main/java/com/thoughtworks/go/config/GoConfigCloner.java

```diff
@@ -36,5 +37,6 @@ public GoConfigCloner() {
                 PipelineNameToConfigMap.class,
                 CachedPluggableArtifactConfigs.class,
                 CachedFetchPluggableArtifactTasks.class);
+        ClonerFactory.applyFixes(this);
```

Comment:
@maheshp This should register the custom `FastCloner`s for `GoConfigCloner`. Tests below to validate.